src/main/java/com/openlattice/users/Auth0Synchronizer.java

```diff
@@ -149,6 +153,8 @@ public void createPrincipal( Auth0UserBasic user, String userId ) {
 
         AclKey userAclKey = spm.lookup( principal );
 
+        organizationService.addMembers( organizationAclKey, ImmutableSet.of( principal ) );
+
         if ( user.getRoles().contains( SystemRole.AUTHENTICATED_USER.getName() ) ) {
```

Comment:
Don't use spm to add principal to principal. Use ```organizationService.addRoletoPrincipalInOrganization```. That way if the logic is ever updated inside of organization this doesn't break.